embeddings/embedding/static/embedding.py

```diff
@@ -36,6 +37,11 @@ def _get_model(self) -> WordEmbeddings:
         return WordEmbeddings(self.name, **self.load_model_kwargs)
 
 
+class StandardStaticWordEmbeddingPL(FlairEmbedding):
```

Comment:
I would add some details why we need seperate class for PL static word embeddings in form of docstring.